frontend/ui/widget/opdsbrowser.lua

```diff
@@ -474,6 +474,8 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     -- download to user selected directory or last opened dir
     local lastdir = G_reader_settings:readSetting("lastdir")
     local download_dir = G_reader_settings:readSetting("download_dir") or lastdir
+    item.author = item.author:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
+    item.title = item.title:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
```

Comment:
While this particular bit of repetition is one thing, the primary reason I proposed it as a util function is that this is hardly the only place where files are being downloaded or written. Of lesser importance is that at least in theory such a utility function could have additional logic related to detecting inferior filesystems.